src/containers/SearchBar/SearchBar.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React from 'react';
```

Comment:
This is really interesting. What does useState do exactly, and what was the reason behind changing it?

Reply:
So `useState` is actually a new feature in React called React Hooks. Hooks allows developers to access React state and lifecycle methods in functional components instead of class components. Basically this means we can use all functional components for our app instead of relying on heavier/slower class components.

Reply:
Awesome! Thank you for that explanation, too!